README.md

```diff
@@ -15,23 +15,17 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
 ## Authors
```

Comment:
- [ ] Please it will be nice to update your `readme file` with a descriptive project topic, and a `short description about the project`, as this will make your `readme file` more informative and professional.